doc/source/dev/roadmap-detailed.rst

```diff
@@ -427,7 +427,13 @@ The following improvements will help SciPy better serve this role.
     - nested ANOVA
     - analysis of covariance (ANCOVA)
```

Comment:
The ANOVA section is always bugging me... To me this is less powerful than things like Sobol'/Shapley indices or moment independent while being more complicated in terms of jargon, etc. (Or I still don't get ANOVA)

Reply:
I will do that when you open the PR 👍